src/about/repl/repl/host/window.js

```diff
@@ -70,7 +70,7 @@ const evalContext =
 
 export const evaluate =
   (id/*:ID*/, code/*:string*/)/*:Task<Never, EvaluationResult>*/ =>
-  Task.future(() => new Promise(resolve => {
+  new Task((succeed, fail) => void(new Promise((resolve, reject) => {
```

Comment:
No it's just type of Task, which is:

```js
class Task <error, value> {
  constructor(perform: (succeed:(value:value) => void, fail:(error:error) => void) => void)
  //...
}
```

So function passed to a constructor must not return non undefined, otherwise type checker will not type check.